src/longRunningCalls/longrunning.ts

```diff
@@ -72,23 +93,33 @@ export class Operation extends EventEmitter {
    * requests.
    */
   constructor(
-    grpcOp: LROOperation,
+    grpcOp: LROOperation | ComputeLROOperation,
```

Comment:
nit: the name `grpcOp` is now quite misleading, please consider renaming.

Reply:
Rename as OperationType